contrib/themes/bogster.toml

```diff
@@ -30,6 +30,7 @@
 
 "ui.background" = { bg = "#161c23" }
 "ui.linenr" = { fg = "#415367" }
+"ui.linenr.selected" = { fg = "#e5ded6" }
```

Comment:
I think we should add a comment saying these are new so that authors can come back and easily change them later.

Reply:
Makes sense; will add them